### PR TITLE
fix: Update report completion status and generate csv button [PT-185257065]

### PR DESCRIPTION
**Pull request by dougmartin**
This updates the completion status to use different text colors depending on the status.  It alss disables the generate csv button until the completion status is succeeded.

### CODE REVIEW

File: researcher-reports/src/components/query-item.tsx [hunk 1]
```diff
@@ -126,6 +126,21 @@ export const QueryItem: React.FC<IProps> = (props) => {
...
+  const succeeded = lowerQueryCompletionStatus === "succeeded";
+  const failed = lowerQueryCompletionStatus === "failed";
+
+  // show the generate button until it succeeds and it is clicked
+  // the button will be disabled until it succeeds
+  const showGenerateCSVLinkButton = !succeeded || !downloadURL;
+
+  const renderCompletionStatus = () => {
+    const suffix = running ? "... (please wait)" : "";
+    const className = running ? "running" : (succeeded ? "succeeded" : (failed ? "failed" : ""));
```
Comment (lublagg) [inline researcher-reports/src/components/query-item.tsx:140]: Just curious, is there a reason not to just use lowerQueryCompletionStatus as the className?

**Review by lublagg — APPROVED** [review]
Looks good 👍  just had a small question about one line

File: researcher-reports/src/components/query-item.tsx (hunk 1 above)
Comment (dougmartin) [reply researcher-reports/src/components/query-item.tsx:140]: Good point :)  There isn't really a reason now that the display code converged on that.  I'll change it - thanks!

Comment (dougmartin) [reply researcher-reports/src/components/query-item.tsx]: @lublagg fixed in https://github.com/concord-consortium/report-service/pull/198/commits/1e2eb0868355f1f815847644a96c77eef905a993

**Pull request merged**